lib/authzHttpClient.js

```diff
@@ -0,0 +1,76 @@
+/*!
+ * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
+ */
+import {httpClient} from '@digitalbazaar/http-client';
+import {httpsAgent} from 'bedrock-https-agent';
+import {isInvalidAccessTokenError} from './errorCheck.js';
+import {getAccessToken, getNewAccessToken} from './oauth2-client.js';
+
+const proxyMethods = new Set([
+  'get', 'post', 'push', 'patch', 'head', 'delete'
```

Comment:
Add `put` to the allowed proxy methods. (It's being added to `http-client` as well).